srcpkgs/xfce4-datetime-plugin/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'xfce4-datetime-plugin'
+pkgname=xfce4-datetime-plugin
+version=0.8.1
+revision=1
+build_style=gnu-configure
+configure_args="--with-locales-dir=/usr/share/locale"
+hostmakedepends="pkg-config intltool"
+makedepends="xfce4-panel-devel"
+depends="xfce4-panel"
```

Comment:
It picks up a bunch of automatic dependencies:

```
   SONAME: libxfce4panel-2.0.so.4 <-> libxfce4panel>=4.12.0_1
   SONAME: libxfce4ui-2.so.0 <-> libxfce4ui>=4.12.1_2
   SONAME: libgtk-3.so.0 <-> gtk+3>=3.0.0_1
   SONAME: libgdk-3.so.0 <-> gtk+3>=3.0.0_1
   SONAME: libpango-1.0.so.0 <-> pango>=1.24.0_1
   SONAME: libgobject-2.0.so.0 <-> glib>=2.68.0_1
   SONAME: libxfce4util.so.7 <-> libxfce4util>=4.12.1_1
   SONAME: libglib-2.0.so.0 <-> glib>=2.68.0_1
   SONAME: libc.so <-> musl>=1.1.24_7
```

does it actually need `xfce4-panel` explicitly as well?

Reply:
Yes, because it requires `libxfce4panel` to build.
I think we can remove it from `depends`, but it's a plugin for the `xfce4-panel` so I'm not sure if there's a way to use without it.

Reply:
Having plugins depend on the application they are a plugin for leads to weird dependency chains; I'd rather avoid that.

Reply:
Removed it from `depends`.